threeML/plugin_prototype.py

```diff
@@ -125,3 +125,14 @@ def inner_fit(self):
         logLike value.
         """
         pass
+
+    @abc.abstractmethod
```

Comment:
Cool. The only issue is that it will crash the AnalysisResults unless a number is returned.